src/libsyntax/ast.rs

```diff
@@ -2605,13 +2605,13 @@ pub type ForeignItem = Item<ForeignItemKind>;
 /// An item within an `extern` block.
 #[derive(Clone, RustcEncodable, RustcDecodable, Debug)]
 pub enum ForeignItemKind {
-    /// A foreign function.
-    Fn(FnSig, Generics, Option<P<Block>>),
-    /// A foreign static item (`static ext: u8`).
+    /// A static item (`static FOO: u8`).
     Static(P<Ty>, Mutability),
-    /// A foreign type.
+    /// A function.
```

Comment:
I disagree with comment changes from this commit from https://github.com/rust-lang/rust/pull/69194#issuecomment-586639669 point of view.

Reply:
Can we deal with this once we have a single `ItemKind` and unified parsing and do the "fan-out" to https://github.com/rust-lang/rust/pull/69194#issuecomment-586639669 at that point? I think that will be less work overall and easier for implementation staging.

Reply:
>Can we deal with this once we have a single ItemKind and unified parsing and do the "fan-out" to #69194 (comment) at that point?

Do you plan to work on that in the near future?

Reply:
I do yes; I figure I will first fix:
> (Work still remains to fuse this with free type aliases, but this can be done later.)

and then the PR after that one will be doing the fan-out + fully fusing the parsing.

Hopefully I should be done this week.

Reply:
Ok, let's land this then.